runbooks/koji/upgrade.md

```diff
@@ -0,0 +1,83 @@
+Koji upgrade
+==============
+
+Issue
+-----
+* You want to upgrade [Koji](http://koji.katello.org) to a newer version
+
+Resolution
+----------
+* Shut down koji services on `koji-fedora28-builder` and `kojibuilder1`.
+  * `koji-fedora28-builder`
```

Comment:
Maybe drop the backticks around the machine names, the rendered version ends up looking a bit odd with how markdown handles these highlighted entries.

Reply:
\o/ markdown handling inconsistencies FTW.  Looked not bad in vscode.  Will remove.